lib/Doctrine/DBAL/Query/QueryException.php

```diff
@@ -36,4 +36,9 @@ public static function nonUniqueAlias($alias, $registeredAliases)
             'in FROM and JOIN clause table. The currently registered ' .
             'aliases are: ' . implode(', ', $registeredAliases) . '.');
     }
+
+    public static function uninitializedTableName(): self
+    {
+        return new self('There is no currently registered table name.');
```

Comment:
That message is misleading, it will be used when there are 2 registered table names. The name of the method is misleading too.

Reply:
Have you a suggestion?